examples/nlp/token_classification/punctuation_capitalization.py

```diff
@@ -206,7 +206,7 @@ def create_pipeline(
         if args.use_weighted_loss_punct:
             logging.info(f"Using weighted loss for punctuation task")
             punct_label_freqs = data_layer.dataset.punct_label_frequencies
-            class_weights = calc_class_weights(punct_label_freqs)
+            class_weights = calc_class_weights(punct_label_freqs, max_id=-1)
```

Comment:
max_id=-1 is the default, please revert the changes in the script

Reply:
When max_id=-1 is passed. It can not handle missing labels. To handle misiing labels, maximum label id should get passed. I put it there to remind us that it does not handle those label missing cases.

Reply:
this script doesn't use descriptor and the dict with labels and labels ids is generated from the training data and passed to the eval layer, I don't think that this max_id is relevant here.

Reply:
Removed it.